### PR TITLE
Delete Size usage #67

### PR DESCRIPTION
**Pull request by lkischler**

### CODE REVIEW

File: Xabe.FFmpeg/Enums/FileExtensions.cs [hunk 1]
```diff
@@ -1,13 +1,9 @@
-﻿using System;
-using System.ComponentModel;
-using System.Reflection;
-
```
Comment (tomaszzmuda) [inline Xabe.FFmpeg/Enums/FileExtensions.cs]: I don't think mp3 or Png is video file

File: Xabe.FFmpeg/Enums/VideoSize.cs [hunk 2]
```diff
@@ -13,18 +13,36 @@ public class VideoSize
...
         ///     VideoSize in FFmpeg format
         /// </summary>
         /// <param name="width">Width of output video</param>
         /// <param name="height">Height of output video</param>
         public VideoSize(int width, int height)
         {
+            Width = width;
+            Height = height;
             Resolution = $"{width}x{height}";
         }
```
Comment (tomaszzmuda) [inline Xabe.FFmpeg/Enums/VideoSize.cs]: I would delete this method and leave only constructor with (int width, int height).
User shoudn't be worried about formatting resolution

File: Xabe.FFmpeg/Enums/VideoSize.cs [hunk 3]
```diff
@@ -13,18 +13,36 @@ public class VideoSize
         /// </summary>
```
Comment (tomaszzmuda) [inline Xabe.FFmpeg/Enums/VideoSize.cs]: I think you should overload ToString to return this property

File: Xabe.FFmpeg/Enums/FileExtensions.cs (hunk 1 above)
Comment (lkischler) [reply Xabe.FFmpeg/Enums/FileExtensions.cs]: I guess it doesn't matter - it is file extensions class not video file extensions.

File: Xabe.FFmpeg/Enums/VideoSize.cs (hunk 3 above)
Comment (lkischler) [reply Xabe.FFmpeg/Enums/VideoSize.cs]: you are right

**Pull request merged**